docs/GettingStarted/configuration/ExporterDeploytime.md

```diff
@@ -2,28 +2,71 @@
 
 The job of the deploy time exporter is to capture the timestamp at which a deployment event happen in a production environment.
 
-In order for proper collection, we require that all deployments associated with a particular application be labelled with the same `app.kubernetes.io/name=<app_name>` label.
+: **NOTE:** In order for proper collection, we require that all deployments associated with a particular application be labelled with the same [APP_LABEL](#app_label), which by default is `app.kubernetes.io/name=<app_name>`.
```

Comment:
include in the end ", where **app_name** is the name of one of the applications being monitored."

Reply:
and add `>` before note

Reply:
slightly adjusted wording, but changed this.